src/content/contributing/index.md

```diff
@@ -10,7 +10,7 @@ sidebar: true
 The ethereum.org website, like Ethereum more broadly, is an open-source project. So if you want to help improve [our portal to Ethereum](/about/), here's how you can help out.
 
 <InfoBanner shouldCenter emoji=":tada:">
-  Claim your POAP token! If you contributed to ethereum.org in 2021, there's a unique POAP waiting for you.{" "}
+  Claim your POAP token! If you contributed to ethereum.org in 2022, there's a unique POAP waiting for you.{" "}
```

Comment:
this can be calculated to avoid hardcoding it, `new Date().getFullYear()` gets the job done

Reply:
This could be an option, but I would push back a little that when the year turns over, we won't necessarily already have POAPs ready for the new year, and thus would suggest we keep this annual (only needs to be updated once a year).

Reply:
I think this should be addressed separately from deploy PR